apps/supernova/src/hooks/useUrlQueryState/utils.ts

```diff
@@ -0,0 +1,70 @@
+const ACTIVE_FILTERS_PREFIX = "f_"
+const ACTIVE_PREDEFINED_FILTER = "predefined_filter"
+const DETAILS_FOR = "details_for"
+const SEARCH_TERM = "search_term"
+const ACTIVE_TAB = "active_tab"
+const SILENCE_REG_EXP = "silence_exp"
+const SILENCE_STATUS = "silence_status"
+const SILENCE_DETAIL = "silence_detail_for"
+
+// This utility extracts filters from the parsed query string
+const transformFiltersToAppState = ({ urlState, filterLabels, prefix }) =>
```

Comment:
As discussed offline. We will introduce tests for this functionality when this is used productive. This is just a POC. Thx!

Reply:
Will be addressed as part of https://github.com/cloudoperators/juno/issues/746